corehq/ex-submodules/casexml/apps/case/fixtures.py

```diff
@@ -28,7 +28,7 @@ def fixture(self):
                 <date_opened/>
                 <last_modified/>
                 <case_property />
-                <index>
+                <index>corehq/ex-submodules/casexml/apps/case/fixtures.py
```

Comment:
looks like this bogey snuck in. 

curious why tests aren't failing....

Reply:
Its in the docstring, which is why tests arent failing, but not sure how it got there.

Reply:
oh whoops i should have realized that.